nixos/tests/gphoto2/defaults.nix

```diff
@@ -0,0 +1,16 @@
+import ../make-test-python.nix (_:
+let
+  machineName = "machine";
+in
+{
+  name = "gphoto2-with-defaults";
+
+  nodes."${machineName}".programs.gphoto2.enable = true;
+
+  testScript = ''
+    ${machineName}.start()
+
+    ${machineName}.succeed("gphoto2 --version")
+    ${machineName}.succeed("getent group camera")
```

Comment:
Also I am skeptical about this test. We don't even check if the program is using the group correctly. So basically the same as above.